openassessment/templates/openassessmentblock/peer/oa_peer_assessment.html

```diff
@@ -66,9 +66,11 @@ <h3 class="peer-assessment__display__title">
                                     </h3>
                                 </header>
 
-                                {% include "openassessmentblock/oa_submission_answer.html" with answer=peer_submission.answer answer_text_label="Your peer's response to the question above:" %}
+                                {% trans "Your peer's response to the question above" as transated_label %}
+                                {% include "openassessmentblock/oa_submission_answer.html" with answer=peer_submission.answer answer_text_label=translated_label %}
```

Comment:
When looking at a peer assessment, I the string "Your peer's response..." is not showing (it's not in the HTML at all).

Note also that the h2 with "The question for this section." in oa_submission_answer.html is not visible. This may not be related to your change-- if it doesn't make sense to fix it as part of your work, please create a bug.